emission/core/wrapper/trip_old.py

```diff
@@ -13,7 +13,7 @@
 
 DATE_FORMAT = "%Y%m%dT%H%M%S-%W00"
 
-class Coordinate:
+class Coordinate(object):
```

Comment:
I am OK with committing this for now, but you should consider using the new datastructures and wrapper classes instead of the old ones.